tests/integration/targets/keycloak_realm_key/tasks/main.yml

```diff
@@ -151,7 +151,8 @@
       - result.end_state.config.enabled == ["true"]
       - result.end_state.config.algorithm == ["RS256"]
       - result.end_state.config.priority == ["140"]
-      - result.msg == "Realm key testkey would be changed: config.priority ['150'] -> ['140']"
+      - >-
+        result.msg == "Realm key testkey would be changed: config.priority ['150'] -> ['140']"
```

Comment:
Then here we keep the `==` operator. Is this the demonstration of many different ways of doing the same thing? Are there rules for using one or the other? Recommendations?

Reply:
This is a module, not a filter. The message is returned as-is from the module.